other/materials_designer/create_point_defect.ipynb

```diff
@@ -9,18 +9,11 @@
     "\n",
```

Comment:
<p>Initialize Configuration and Builder parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>